be/CMakeLists.txt

```diff
@@ -441,12 +441,13 @@ check_function_exists(sched_getcpu HAVE_SCHED_GETCPU)
 
 # compiler flags that are common across debug/release builds
 #  -Wall: Enable all warnings.
-#  -Wno-sign-compare: suppress warnings for comparison between signed and unsigned
 #    integers
 #  -pthread: enable multithreaded malloc
 #  -DBOOST_DATE_TIME_POSIX_TIME_STD_CONFIG: enable nanosecond precision for boost
 #  -fno-omit-frame-pointers: Keep frame pointer for functions in register
-set(CXX_COMMON_FLAGS "${CXX_COMMON_FLAGS} -Wall -Wno-sign-compare -pthread -Werror")
+set(CXX_COMMON_FLAGS "${CXX_COMMON_FLAGS} -Wall -Wextra -pthread -Werror")
```

Comment:
```suggestion
set(CXX_COMMON_FLAGS "${CXX_COMMON_FLAGS} -Wall -Wextra -pthread -Werror -pedantic")
```

Reply:
Can add an extra option.

Reply:
I tried pedantic, still many compilation warnings, maybe these can be done in other pr after

Reply:
> I tried pedantic, still many compilation warnings, maybe these can be done in other pr after

Okey, it is a great job.